llvm/test/Analysis/CtxProfAnalysis/flatten-check-path.ll

```diff
@@ -0,0 +1,85 @@
+; REQUIRES: asserts
+; Check that the profile annotator works: we hit an exit and non-zero paths to
+; already visited blocks are not disconsidered.
```

Comment:
Not sure what you are trying to say by "are not disconsidered" here, especially with the double-negative.